ios_ui/MovieRec/MovieImages.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MovieImages.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/20/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
```

Comment:
Random thought, but architecturally, I feel like this can be a lot cleaner, we can talk it over at lunch sometime lol. =)